docs/docs/spec.md

```diff
@@ -334,7 +331,7 @@ locations listed below:
 
 To determine whether a type usage appears in a null-marked scope:
 
-Look for an `@org.jspecify.nullness.NullMarked` annotation on any of the scopes
+Look for a `@NullMarked` annotation on any of the scopes
```

Comment:
I think there is value is fully specifying the name to be clear that we're not matching on simple name. Ditto in a couple places below.

Alternatively, I can see an argument that we could define somewhere that, when we say "`@NullMarked," etc., we _always_ mean that specific class.

But also, for fun, I'm choosing to interpret this comment as another look of disapproval toward the current package name :)

Reply:
Ah, I read "details common to all annotations" as defining just that. Could we tweak it so it accomplishes that more clearly? I think the FQN is obtrusive and of course is just one more place to fix on rename (which obvs I expect to happen SEVERAL MORE TIMES)

Reply:
tweaked that section.

Reply:
Thanks, I think that helps.